.github/workflows/graalwasm-micronaut-excelice.yml

```
@@ -0,0 +1,48 @@
+name: Test GraalWasm Micronaut Excelice Demo
+
+on:
+  push:
+    paths:
+      - 'graalwasm/graalwasm-micronaut-excelice/**'
+      - '.github/workflows/graalwasm-micronaut-excelice.yml'
+  pull_request:
+    paths:
+      - 'graalwasm/graalwasm-micronaut-excelice/**'
+      - '.github/workflows/graalwasm-micronaut-excelice.yml'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  run:
+    name: 'graalwasm-micronaut-excelice'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          java-version: '21'
```

Please use the latest release to be consistent with all other demos:
```suggestion
          java-version: '24.0.0'
```